kitsune/search/v2/signals/forums.py

```diff
@@ -0,0 +1,25 @@
+from django.db.models.signals import post_save, post_delete
+from django.dispatch import receiver
+from django.conf import settings
+from kitsune.search.v2.es7_utils import index_object, delete_object, index_objects_bulk
+from kitsune.forums.models import Thread, Post
+
+
+@receiver(post_save, sender=Thread)
+def handle_forum_thread_save(instance, **kwargs):
+    if settings.ES_LIVE_INDEXING:
+        index_objects_bulk.delay(
+            "ForumDocument", list(instance.post_set.values_list("pk", flat=True))
```

Comment:
why the list here?

Reply:
this has other occurrences too.

Reply:
Celery won't serialize a `values_list` or cast it to a list itself, it needs to be done manually. (You'll see the error if you remove the `list()` and run the tests with `CELERY_TASK_ALWAYS_EAGER=False`.)